hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -156,10 +156,16 @@ private Constants() {
   public static final String MAXIMUM_CONNECTIONS = "fs.s3a.connection.maximum";
   public static final int DEFAULT_MAXIMUM_CONNECTIONS = 96;
 
-  // Expiration time of s3 http connection from the connection pool.
-  // See {@code com.amazonaws.ClientConfiguration#setConnectionTTL}
+  /**
+   * Configuration option to configure expiration time of
+   * s3 http connection from the connection pool in milliseconds: {@value}.
+   */
   public static final String CONNECTION_TTL = "fs.s3a.connection.ttl";
-  public static final long DEFAULT_CONNECTION_TTL = ClientConfiguration.DEFAULT_CONNECTION_TTL;
+
+  /**
+   * Default value for {@value CONNECTION_TTL}: {@value}.
```

Comment:
should be @link or @code, surely

Reply:
It will convert it to the actual value so I think it is good https://javadoc.io/doc/org.apache.hadoop/hadoop-aws/latest/index.html